src/main/java/step2/LottoMain.java

```diff
@@ -0,0 +1,30 @@
+package step2;
+
+import step2.model.LottoGame;
+import step2.model.LottoTotalCalculator;
+import step2.model.WinnerTier;
+import step2.util.LottoStringtoNumbers;
+import step2.view.LottoInput;
+import step2.view.LottoOutput;
+import java.util.List;
+import java.util.Map;
+
+public class LottoMain {
+    public static void main(String[] args) {
+        int lottoMoney = LottoInput.inputGameMoney();
+
+        LottoGame lottoGame = new LottoGame(lottoMoney);
+
+        LottoOutput.printPurchaseNumbers(lottoGame.getLottoCount());
+        LottoOutput.printLottoNumbers(lottoGame);
+
+        String inputWinnerNumbers = LottoInput.inputWinnerNumbers();
+        List<Integer> WinnerNumbers = LottoStringtoNumbers.convertStringtoNumbers(inputWinnerNumbers);
```

Comment:
당첨 번호도 결국 Lotto이지 않을까 합니다. 해당 코멘트와 같이 고민해보아요 :)
```suggestion
        Lotto winningLotto = LottoInput.inputWinnerNumbers();
```

Reply:
변경 하여 반영하였습니다.